sale_import_base/datamodels/sale_order.py

```diff
@@ -0,0 +1,22 @@
+from odoo.addons.datamodel import fields
+from odoo.addons.datamodel.datamodels.base import BaseDatamodel
+
+FIELDS_REQUIRED_address_customer = ["email"]
+
+
+class SaleOrderDatamodel(BaseDatamodel):
+    _name = "sale.order"
+
+    address_customer = fields.NestedModel("sale.order.address", required=True)
+    address_shipping = fields.NestedModel("sale.order.address", required=True)
+    address_invoicing = fields.NestedModel("sale.order.address", required=True)
+    lines = fields.NestedModel("sale.order.line", many=True, required=True)
+    amount = fields.NestedModel("sale.order.amount", required=True)
+    payment_mode = fields.Str()
+    transaction_id = fields.Integer()
+    status = fields.Str()
```

Comment:
I do not think that we need this key here. For importing done order (migration process) I thinks it's better to have a separated module. As it's more a one shot module that we will uninstall after.
We can create a sale_import_base_migration will all the logic of validating the picking, refunding the invoice